Bottom: 14e61c02726b6bf48271d2ae23ac43d5f2fd66cd
Top:    64c8b480ec113d1b5187b073d18749861c484641
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-05 23:09:55 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 9b2f76e..296eeeb 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
